<launch>
  <node pkg="nao_pose" type="pose_manager.py" name="pose_manager">
    <rosparam file="$(find nao_pose)/config/basic_poses.yaml" command="load" ns="poses"/>
  </node>
</launch>
